go-controller/pkg/node/gateway_udn.go

```diff
@@ -659,23 +665,36 @@ func (udng *UserDefinedNetworkGateway) getV6MasqueradeIP() (*net.IPNet, error) {
 // 2000:   from all to 169.254.0.12 lookup 1007
 // 2000:   from all fwmark 0x1002 lookup 1009
 // 2000:   from all to 169.254.0.14 lookup 1009
+// If isPodNetworkAdvertised is set to true then we add following IP rules
+// for 10.132.0.0/14 UDN subnet
+// 2000:	from all fwmark 0x1001 lookup 1009
+// 2000:	from all to 10.132.0.0/14 lookup 1009
```

Comment:
`add` is confusing here. It can be understood as added to the rules above. I would explain it better.